Spring-server/src/main/java/ikpi63holding/postclient/jwt/filters/JwtAuthenticationFilter.java

```diff
@@ -25,6 +28,8 @@ public JwtAuthenticationFilter(AuthenticationManager authenticationManager,
             JwtProperties jwtProperties) {
         this.authenticationManager = authenticationManager;
         this.jwtProperties = jwtProperties;
+        // We can set login endpoint here
+        //setRequiresAuthenticationRequestMatcher(new AntPathRequestMatcher("/logme", "POST"));
```

Comment:
We can setup out endpoint like this, pulled it out of abstract class underneath.